app/src/test/scala/org/alephium/explorer/persistence/queries/TokenQueriesSpec.scala

```diff
@@ -82,22 +83,35 @@ class TokenQueriesSpec extends AlephiumFutureSpec with DatabaseFixtureForEach wi
       }
     }
 
-    // TODO Test data aren't coherent, we currently only test that query doesn't throw an error -_-'
     "list address tokens with balance" in {
       implicit val groupSetting: GroupSetting = GroupSetting(4)
       val testData     = Gen.nonEmptyListOf(blockAndItsMainChainEntitiesGen()).sample.get
       val inputs       = testData.flatMap(_._1.inputs)
       val tokenOutputs = testData.map(_._4)
       val addresses    = tokenOutputs.map(_.address)
-      val pagination   = Pagination.unsafe(1, 10)
+      val pagination   = Pagination.unsafe(1, 100)
+      val now          = TimeStamp.now()
```

Comment:
Does we test the correctness of `lockedBalance`?

Reply:
yes, I checked data generated with scalacheck and it produce both locked and unlocked tokens